kalohq/ui
# Renamed css modules files

Uses the `.module.css` suffix to the CSS modules.

This is making consumable the package by any `create-react-app` and `gatsby` site without a custom webpack build to handle the CSS modules.

It makes the style guides use the styles created by the `gulp` bundle instead to the custom one inside `gatsby-node`. In this way, the vanilla CSS used by the style guides is 1:1 with the one built by the package.

In addition to this, the peer dependencies have been aligned to the module used.

### How to test it
1. Check out this branch
1. `npm install`
1. `npm run start-styleguide`
1. Check that everything in the style guides looks good
